python/sort_animals.py

```diff
@@ -0,0 +1,13 @@
+class Animal:
+    def __init__(self, name, number_of_legs):
+        self.name = name
+        self.number_of_legs = number_of_legs
```

Comment:
This isn't used here, looks like a stub for testing. Move it into the test file.